content/javascript/concepts/storage/terms/clear.md

```diff
@@ -0,0 +1,54 @@
+---
+Title: '.clear()'
```

Comment:
A folder named clear should be added inside "terms". The path should be -
content/javascript/concepts/storage/terms/clear/clear.md